animations and styles

dianadeayer opened the pull request:
![image](https://github.com/user-attachments/assets/1e2345df-c500-4318-80d4-367f08172ec2)

I finished the general body.
Next steps include: create animations and adjust sizes.

dianadeayer committed:
html body

dianadeayer committed:
animations completed

dianadeayer commented:
![image](https://github.com/user-attachments/assets/306089aa-a295-45e8-8c39-ffedf9b463ef)

Animations and box shadow hover are now completed.

josuemonroy2023 commented:
It looks good but there is room for improvement.
First the animations are good but those should have a transition, to improve the user experience.
After that when you stop the wiggle it looks weird that it return to the initial position, it should stop at the same point that the user hover at.
Then we talk about ARIA properties, make sure tho use them.
Next you can use variables for colors and other common values.
Finally you can change the table, instead of applying changes to the table is better to add a class to the table and apply styles there.

dianadeayer committed:
fixed button animation and added aria properties

dianadeayer commented:
![image](https://github.com/user-attachments/assets/e669660e-5c42-4739-8721-1fb875c762d7)
I fixed the button animation, changed the table, and added aria properties to my table and div that contains the progress indicator. I'm still reading about them, and I have some questions. I read this page to learn more: [](https://www.w3.org/WAI/ARIA/apg/patterns/) , and it's been useful, but I'm still a little bit confused. I made a list of my questions so we can talk about it on Friday.

dianadeayer commented:
https://www.w3.org/WAI/ARIA/apg/patterns/

dianadeayer committed:
fixed table styles and buttons transitions

dianadeayer commented:
![image](https://github.com/user-attachments/assets/0f746720-ec76-4e70-b4f9-1d57e4d01f4b)

Added box shadow to the table, transition to the hover button and centered tittles.
